templates/3a_monitor_MIBI_run.ipynb

```diff
@@ -111,10 +111,11 @@
    "source": [
```

Comment:
<p>Add a description for what this callback does in the documentation above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>